build_runner/test/generate/build_definition_test.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:convert';
+
+import 'package:build/build.dart';
+import 'package:logging/logging.dart';
+import 'package:test/test.dart';
+import 'package:watcher/watcher.dart';
+
+import 'package:build_runner/build_runner.dart';
+import 'package:build_runner/src/asset_graph/graph.dart';
+import 'package:build_runner/src/asset_graph/node.dart';
+import 'package:build_runner/src/generate/build_definition.dart';
+import 'package:build_runner/src/generate/options.dart';
+import 'package:build_runner/src/package_graph/package_graph.dart';
+import 'package:build_runner/src/util/constants.dart';
+
+import '../common/common.dart';
+
+main() {
+  group('load', () {
+    Map<AssetId, DatedValue> assets;
+    BuildOptions options;
+
+    setUp(() {
+      assets = <AssetId, DatedValue>{};
+      var assetReader = new InMemoryRunnerAssetReader(assets, 'a');
+      var assetWriter = new InMemoryRunnerAssetWriter();
+      var packageA = new PackageNode('a', null, null, null);
+      var packageGraph = new PackageGraph.fromRoot(packageA);
+      options = new BuildOptions(
+          reader: assetReader,
+          writer: assetWriter,
+          packageGraph: packageGraph,
+          logLevel: Level.OFF,
+          writeToCache: true);
+    });
+
+    group('updates', () {
```

Comment:
[nit] Using `updates` as the group name makes a lot of these tests read weird. Maybe use, `For build updates`?

Reply:
`updates` is a field on `BuildDefinition` and this is the group of tests that are testing that field